src/components/app/App.jsx

```diff
@@ -23,9 +23,7 @@ import { ToastsProvider, Toasts } from '../Toasts';
 import EnrollmentCompleted from '../executive-education-2u/EnrollmentCompleted';
 
 // Create a query client for @tanstack/react-query
-const queryClient = new QueryClient({
-  refetchOnWindowFocus: false,
```

Comment:
This setting was improperly configured, so it wasn't really ever working. Which is actually a good thing; it enables automatically re-fetching the redeemability of content with [the enterprise-access `can-redeem` API](https://enterprise-access.edx.org/api/schema/redoc/#tag/Subsidy-Access-Policy-Redemption/operation/api_v1_policy_enterprise_customer_can_redeem_list).

The setting is set to true by default in `@tanstack/react-query`, and for good reason; it helps ensure users are always working with the latest/fresh data. We should structure our use of `useQuery` in a way that allows us to take advantage of this feature, as is shown here:

https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/2828721/e2d2969a-d901-47e6-8560-bfb617192111

Reply:
:woah: